app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/Wishlist/Grid/Renderer/Description.php

```diff
@@ -23,6 +23,7 @@ class Mage_Adminhtml_Block_Customer_Edit_Tab_Wishlist_Grid_Renderer_Description
 {
     public function render(Varien_Object $row)
     {
-        return nl2br(htmlspecialchars($row->getData($this->getColumn()->getIndex())));
+        $value = $row->getData($this->getColumn()->getIndex());
+        return is_string($value) ? nl2br(htmlspecialchars($value)) : '';
```

Comment:
This one modifies the original behavior. For example, if `$value` is an array, no error is thrown, making debug difficult. As our intention is to fix null deprecation in PHP8.1, we have to be careful if we are doing more than that. Is the side effect of doing more desirable? In this case, no errors are suppressed, is it what we want?